<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EbDialogChangeHead</class>
 <widget class="QDialog" name="EbDialogChangeHead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonDefault">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCustom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>custom</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetDefaultHeads">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open file</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelImageFile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>image file</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImageResult">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>image result</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImageFileTip">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>image file tip</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImageResultTip">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>image result tip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
